example/src/TopBarTextExample.js

```diff
@@ -48,6 +48,7 @@ export default class TopBarTextExample extends Component {
         pressColor='rgba(0, 0, 0, .2)'
         indicatorStyle={styles.indicator}
         style={styles.tabbar}
+        labelStyle={{ color: '#fbdc08', fontWeight: '500' }}
```

Comment:
Move this to the `StyleSheet.create` call. Also probably just make it `{ color: '#fff', fontWeight: '500' }` since I want to show that the default styles are based on material design.